ssl/ssl_cert.c

```diff
@@ -306,18 +328,18 @@ int ssl_cert_add1_chain_cert(SSL_CONNECTION *s, SSL_CTX *ctx, X509 *x)
 
 int ssl_cert_select_current(CERT *c, X509 *x)
 {
-    int i;
+    size_t i;
     if (x == NULL)
```

Comment:
Blank line again.  Yeah, preexisting but good to address it when changes happen.

Reply:
found some more & changed them too.

Reply:
I noticed some others, but had decided to ceased commenting....